packages/office-ui-fabric-react/src/components/Link/Link.tsx

```diff
@@ -48,7 +48,15 @@ export class Link extends BaseComponent<ILinkProps, any> implements ILink {
             onClick={ this._onClick }
             ref={ this._resolveRef('_link') }
           >
-            { children }
+            <a href=""
```

Comment:
The desired color is automagically applied to the href. Will keep looking for a way to trigger the desired color for the button.